_tasks/test_examples.ts

```diff
@@ -0,0 +1,67 @@
+import { Buffer, readLines } from "../deps/std/io.ts"
+import * as path from "../deps/std/path.ts"
+import { writeAll } from "../deps/std/streams.ts"
+import { assert } from "../deps/std/testing/asserts.ts"
+
+if (!Deno.args[0]) {
+  throw new Error("specify examples directory as first CLI argument")
+}
+
+const examplesDir = path.join(Deno.cwd(), Deno.args[0])
```

Comment:
No need to explicitly join this with `cwd`, this will happen implicitly.
```suggestion
const examplesDir = Deno.args[0]
```

Reply:
:heavy_check_mark: